app/src/test/java/org/oppia/android/app/activity/route/ActivityRouterModuleTest.kt

```diff
@@ -109,36 +115,19 @@ class ActivityRouterModuleTest {
   }
 
   @Test
-  fun testInjectActivityRouterModule_routeToRecentlyPlayedActivity() {
-    activityRule.scenario.onActivity { activity ->
-      assert(
-        destinationRoutes.containsKey(
-          DestinationScreen.DestinationScreenCase.RECENTLY_PLAYED_ACTIVITY_PARAMS
-        )
-      )
-      destinationRoutes[DestinationScreen.DestinationScreenCase.RECENTLY_PLAYED_ACTIVITY_PARAMS]
-        .let { route ->
-          val intent = route?.createIntent(
-            activity,
-            DestinationScreen.getDefaultInstance()
-          )
-          assertThat(intent).hasComponentClass(RecentlyPlayedActivity::class.java)
-        }
-    }
+  fun testInjectDestinationRoutes_hasRouteForRecentlyPlayedActivity() {
+    assertThat(
+      destinationRoutes.containsKey(DestinationScreenCase.RECENTLY_PLAYED_ACTIVITY_PARAMS)
+    ).isTrue()
```

Comment:
```suggestion
    assertThat(destinationRoutes).containsKey(DestinationScreenCase.RECENTLY_PLAYED_ACTIVITY_PARAM)
```

This is a more idiomatic way to do this.

Reply:
Updated